.travis.yml

```diff
@@ -59,7 +59,14 @@ install:
     - sudo apt-get install cmake
 
     # Install Mongo C Driver
-    - pushd mongo-c-driver; ./autogen.sh --enable-tests=no --enable-examples=no --with-libbson=bundled; make; sudo make install; popd;
+    - pushd mongo-c-driver
+
+    # TODO: Update this to our real minimum for the C++11 driver 3.1 release, once known.
+    - git checkout master
+
+    - ./autogen.sh --enable-tests=no --enable-examples=no --with-libbson=bundled; make; sudo make install
```

Comment:
Eh, maybe. We only *really* need that when doing the ASAN builds, and we don't do those on travis. Maybe it is better to ensure that the driver tests pass when the C driver is built both ways? In the real world, the C++11 driver will encounter both C driver modes, at least for now.